test/fixtures/basic/package.json

```diff
@@ -4,6 +4,7 @@
     "run-series": "^1.1.1"
   },
   "devDependencies": {
+    "coffee-script": "^1.10.0",
```

Comment:
Don't we already have several `devDependencies` that add to `.bin` (e.g. asar, extract-zip, mkdirp...)?  Still seems unnecessary.